src/parser/mod.rs

```diff
@@ -273,6 +274,7 @@ enum ParserState {
     ConnectBy,
 }
 
+#[cfg_attr(feature = "std", derive(Debug))]
```

Comment:
can we remove the introduced Debug statements if they're unrelated to the PR?

Reply:
Removed.